kubernetes/values.yaml

```diff
@@ -1,56 +1,214 @@
+    limits:
+      cpu: 100m
+      memory: 256Mi
+    requests:
+      cpu: 100m
+      memory: 256Mi
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
 parser:
+  replicaCount: 1
   image:
     repository: kubevious/parser
     tag: 0.7.7
     pullPolicy: IfNotPresent
-  name: kubevious-parser
-  port: 4002
-  cpu: 100m
-  memory: 200Mi
+  imagePullSecrets: []
+  podAnnotations: {}
+
+  podSecurityContext: {}
+    # fsGroup: 2000
+
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+
+  service:
+    type: ClusterIP
+    port: 4002
+  resources:
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    limits:
+      cpu: 100m
+      memory: 256Mi
+    requests:
+      cpu: 100m
+      memory: 256Mi
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+
+  serviceAccount:
+    # Specifies whether a service account should be created
+    create: true
+    # Annotations to add to the service account
+    annotations: {}
+    # The name of the service account to use.
+    # If not set and create is true, a name is generated using the fullname template
+    name: ""
 ui:
+  replicaCount: 1
   image:
     repository: kubevious/ui
     tag: 0.7.17
     pullPolicy: IfNotPresent
-  name: kubevious-ui
-  svcName: kubevious-ui-svc
-  port: 80
-  cpu: 25m
-  memory: 100Mi
+  imagePullSecrets: []
+  podAnnotations: {}
+
+  podSecurityContext: {}
+    # fsGroup: 2000
+
+  securityContext: {}
+    # capabilities:
+    #   drop:
+    #   - ALL
+    # readOnlyRootFilesystem: true
+    # runAsNonRoot: true
+    # runAsUser: 1000
+
+  service:
+    type: ClusterIP
+    port: 80
+  resources:
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    limits:
+      cpu: 25m
+      memory: 128Mi
+    requests:
+      cpu: 25m
+      memory: 128Mi
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
 mysql:
   image:
     repository: mysql
     tag: 8.0.19
     pullPolicy: IfNotPresent
-  name: kubevious-mysql
-  svcName: kubevious-mysql-svc
-  port: 3306
-  cpu: 250m
-  memory: 1000Mi
-  storage: 20Gi
-  storageClass: null
-  db: kubevious
-  user: root
-  pass: ""
-cluster:
-  domain: cluster.local
+  db_name: kubevious
+  db_user: "root"
+  db_password: ""
+  root_password: ""
```

Comment:
db root password is required unless you specify the allow empty password, which is not the best approach, i think... So mysql user should be something different to "root" but in case you do not provide a root password the chart will generate a random one.